lefthook.yaml

```
@@ -30,15 +30,15 @@
         pnpm fix_code_quality
         git add {staged_files}
     8_check_type_errors:
-      run: pnpm check_type_errors     
-    9_check_tests:
-      run: |
-        set -e
-        pnpm check_tests   
+      run: pnpm check_type_errors       
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)